### ISSUE TITLE
Managing out-of-date official doc translations 

### ISSUE THREAD

**Issue by tokusumi** [fastapi/fastapi#5642]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Description

* For contributing translations, it helps that tools or notification to Inform us out of dates translations.
* (I believe) translated documents should shows if up-to-date or out-of-dates to original document. it help users.

### Wanted Solution

First of all, to solve how to search out-of-date translations, I created simple tools ([PR](#5641)) to compare committed date between original and translated docs. It works (but not correct).

How about merging the tools and collaborate them to notify it for developer and user with something like GitHub Actions?

### Alternatives

* adding revision number in each documents and parse it for developer. this way has advantages as for user to check if up-to-date or not when reading.

**Comment by mbroton** [comment]
> (I believe) translated documents should shows if up-to-date or out-of-dates to original document. it help users.

Agree! Also, it could be misleading, if there's no such an information.
I'm not sure if in the form you suggested, but It would be great to have such a functionality.

**Comment by codespearhead** [comment]
> I created simple tools ([PR](https://github.com/tiangolo/fastapi/issues/5642#5641)) to compare committed date between original and translated docs

Can you share the link to those tools with us? It seems that the link in the "PR" is wrong.

Also, it seems to me that it's more of a MkDocs localization problem than a FastAPI's.

From what I could gather, there's no official way to handle outdated translations in MkDocs or ReadTheDocs 😬.

**Comment by YuriiMotov** [comment]
I think this was solved (in a bit different way) by: https://github.com/fastapi/fastapi/pull/13933
